testng-core-api/src/main/java/org/testng/xml/XmlTest.java

```diff
@@ -663,23 +663,4 @@ public XmlGroups getXmlGroups() {
   public boolean nameMatchesAny(List<String> names) {
     return names.contains(getName());
   }
-
-  /**
-   * @param xmlTest - The {@link XmlTest} that represents the current <code>test</code>
-   * @return - <code>true</code> if group based selection was employed and false otherwise.
-   */
-  public static boolean isGroupBasedExecution(XmlTest xmlTest) {
```

Comment:
It's a public api. Maybe deprecated it instead of remove it if it was already released once.

Reply:
This was introduced by me in `7.5-SNAPSHOT` and its not public yet. That's why I directly went ahead and deleted it.